code/core/package.json

```diff
@@ -32,74 +32,16 @@
     }
   },
   "exports": {
-    ".": {
-      "types": "./dist/index.d.ts",
-      "import": "./dist/index.js",
-      "require": "./dist/index.cjs"
-    },
-    "./internal/core-server": {
-      "types": "./dist/core-server/index.d.ts",
-      "import": "./dist/core-server/index.js",
-      "require": "./dist/core-server/index.cjs"
-    },
-    "./internal/core-server/presets/common-preset": {
-      "import": "./dist/core-server/presets/common-preset.js",
-      "require": "./dist/core-server/presets/common-preset.cjs"
-    },
-    "./internal/core-server/presets/common-override-preset": {
-      "import": "./dist/core-server/presets/common-override-preset.js",
-      "require": "./dist/core-server/presets/common-override-preset.cjs"
-    },
```

Comment:
these are no longer exported but referenced directly within storybook core. I don't see any reason to "expose" these to the outside.